temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -229,3 +230,28 @@ message GetRegionResponse {
     // The temporal cloud region.
     temporal.api.cloud.region.v1.Region region = 1;
 }
+
+message GetAccountRequest {
+}
+
+message GetAccountResponse {
+    // The account.
+    temporal.api.cloud.account.v1.Account account = 1;
+}
+
+message UpdateAccountRequest {
+    // The updated account specification to apply.
+    temporal.api.cloud.account.v1.AccountSpec spec = 1;
+    // The version of the account for which this update is intended for.
+    // The latest version can be found in the GetAccount operation response.
+    string resource_version = 2;
```

Comment:
Doesn't matter much, but was a little offputting to see the order of these two fields is different than it is in the account object itself